granary/facebook.py

```diff
@@ -579,6 +580,9 @@ def _create(self, obj, preview=None, include_link=False, ignore_formatting=False
           error_plain='No content text found.',
           error_html='No content text found.')
 
+    if obj.get('displayName') and mf2util.is_name_a_title(unicode(obj.get('displayName')), unicode(obj.get('content'))):
+        content = obj.get('displayName') + u"\n\n" + content
```

Comment:
you'll need to handle `ignore_formatting` here. the easiest way might actually be to put this logic into `Source._content_for_create()`. twitter overrides that, and does pass through, but i think it might still work ok. if its tests pass, i'd be ok with it.